src/components/Ootd/OotdLayoutPreview.tsx

```
@@ -60,17 +71,21 @@ export function OotdLayoutPreview({
   const maxY = Math.max(...boundsList.map((b) => b.maxY));
   const layoutW = maxX - minX;
   const layoutH = maxY - minY;
-  const scale = Math.min(
+  const scale = Math.max(
     width / Math.max(layoutW, 1),
     height / Math.max(layoutH, 1)
   );
```

ESLint

[error] Replace `⏎····width·/·Math.max(layoutW,·1),⏎····height·/·Math.max(layoutH,·1)⏎··` with `width·/·Math.max(layoutW,·1),·height·/·Math.max(layoutH,·1)`

(prettier/prettier)